Description: Remove traceback2 requirements
 We can conveniently use system's traceback lib instead.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2021-12-29

--- unittest2-1.1.0.orig/setup.py
+++ unittest2-1.1.0/setup.py
@@ -57,7 +57,7 @@ KEYWORDS = "unittest testing tests".spli
 # Both install and setup requires - because we read VERSION from within the
 # package, and the package also exports all the APIs.
 # six for compat helpers
-REQUIRES = ['six>=1.4', 'traceback2'],
+REQUIRES = ['six>=1.4'],
 
 params = dict(
     name=NAME,
--- unittest2-1.1.0.orig/unittest2/case.py
+++ unittest2-1.1.0/unittest2/case.py
@@ -7,7 +7,10 @@ import difflib
 import logging
 import pprint
 import re
-import traceback2 as traceback
+try:
+    import traceback
+except ImportException:
+    import traceback2 as traceback
 import types
 import unittest
 import warnings
--- unittest2-1.1.0.orig/unittest2/result.py
+++ unittest2-1.1.0/unittest2/result.py
@@ -4,7 +4,10 @@ import sys
 import unittest
 
 from six.moves import StringIO
-import traceback2 as traceback
+try:
+    import traceback
+except ImportError:
+    import traceback2 as traceback
 
 from unittest2 import util
 from unittest2.compatibility import wraps
--- unittest2-1.1.0.orig/unittest2/test/test_result.py
+++ unittest2-1.1.0/unittest2/test/test_result.py
@@ -1,6 +1,9 @@
 import sys
 import textwrap
-import traceback2 as traceback
+try:
+    import traceback
+except ImportError:
+    import traceback2 as traceback
 
 import six
 from six.moves import StringIO
